src/lib/api/base.js

```diff
@@ -29,6 +31,19 @@ const responseRejectInterceptor = error => {
     goTo(`${AuthenticationRoutes.login}?sessionExpired`);
   }
 
+  errorsList.forEach(errorCode => {
+    if (
+      error.response &&
+      error.response.status === errorCode &&
+      !urlShouldNotRedirect(error.response.request.responseURL)
+    ) {
+      const link = error.response.data.error_id
+        ? `${FrontSiteRoutes.errors}?errorCode=${errorCode}&errorId=${error.response.data.error_id}`
```

Comment:
why call it `errorId` since it is the `sentryId`?

Reply:
I asked myself the same question, but since we call it `errorId` in the [backend](https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/errors.py#L44) I just thought we should stick to it and IMHO i think it's descriptive enough (maybe I am wrong)